src/Functions/match.cpp

```diff
@@ -20,6 +20,7 @@ using FunctionMatch = FunctionsStringSearch<MatchImpl<NameMatch, false>>;
 void registerFunctionMatch(FunctionFactory & factory)
 {
     factory.registerFunction<FunctionMatch>();
+    factory.registerAlias("REGEXP_MATCHES", NameMatch::name);
```

Comment:
We should make alias case insensitive

Reply:
Function name in PostgreSQL is case sensitive, is it better to keep consistent with PostgreSQL in ClickHouse ?